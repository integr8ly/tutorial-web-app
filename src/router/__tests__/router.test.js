import React from 'react';
import { shallow } from 'enzyme';
import Router from '../router';

describe('Router Component', () => {
  it('should render a basic component', () => {
    const component = shallow(<Router />);

    expect(component).toMatchSnapshot();
  });
});
