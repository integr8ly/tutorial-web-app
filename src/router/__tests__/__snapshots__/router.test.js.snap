// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Router Component should render a basic component 1`] = `
<div>
  <Switch>
    <Route
      component={[Function]}
      exact={true}
      key="/"
      path="/"
    />
    <Route
      component={[Function]}
      exact={true}
      key="/home"
      path="/home"
    />
    <Route
      component={[Function]}
      exact={true}
      key="/settings"
      path="/settings"
    />
    <Route
      component={[Function]}
      exact={true}
      key="/dev-resources"
      path="/dev-resources"
    />
    <Route
      component={[Function]}
      exact={true}
      key="/tutorial/:id"
      path="/tutorial/:id"
    />
    <Route
      component={[Function]}
      exact={false}
      key="/tutorial/:id/task/:task/:step?"
      path="/tutorial/:id/task/:task/:step?"
    />
    <Route
      component={[Function]}
      exact={true}
      key="/oauth/callback"
      path="/oauth/callback"
    />
    <Route
      component={[Function]}
      exact={false}
      key="/congratulations/:id"
      path="/congratulations/:id"
    />
    <Route
      component={[Function]}
      exact={false}
      key="*"
      path="*"
    />
    <Redirect
      from="/"
      push={false}
      to="/home"
    />
  </Switch>
</div>
`;
